akka-stream-tests/src/test/java/akka/stream/javadsl/FlexiMergeTest.java

```diff
@@ -7,6 +7,7 @@
 import java.util.List;
 import java.util.HashSet;
 
+import akka.stream.javadsl.japi.Function;
```

Comment:
is this needed? Assuming that the tests were working before, this is an unused import.